OpenOversight/tests/test_functional.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+
+def test_user_can_load_homepage_and_get_to_form(mockdata, browser):
+    browser.get("http://localhost:5000")
+
+    # Complainant loads homepage
+    assert "OpenOversight" in browser.title
+    browser.find_element_by_link_text("Try it").click()
+    time.sleep(5)
```

Comment:
Explicit sleeps is a selenium antipattern; it makes the suite slower than it needs to be and brittle and doesn't solve the problem you want solved. Check http://stackoverflow.com/a/26567563 for options in how to do it in python. Basically, waiting until an element is visible, or dying if it's not visible in x seconds.